test/liblll/EndToEndTest.cpp

```diff
@@ -583,24 +586,34 @@ BOOST_AUTO_TEST_CASE(allgas)
 
 BOOST_AUTO_TEST_CASE(send_two_args)
 {
-	char const* sourceCode = R"(
-		(returnlll
-			(send 0xdead 42))
-	)";
-	compileAndRun(sourceCode);
-	callFallbackWithValue(42);
-	BOOST_CHECK(balanceAt(Address(0xdead)) == 42);
+	// "send" does not retain enough gas to be able to pay for account creation.
+	// Disabling for non-tangerineWhistle VMs.
+	if (dev::test::Options::get().evmVersion().canOverchargeGasForCall())
+	{
+		char const* sourceCode = R"(
+			(returnlll
+				(send 0xdead 42))
```

Comment:
Eventually `send` and `msg` should be updated I think.

@benjaminion @zigguratt what do you think?